docs/modules/ROOT/pages/how-tos/creating.adoc

```diff
@@ -1 +1,131 @@
-= Creating applications and components
\ No newline at end of file
+= Creating applications and components
+
+To create an application in {ProductName}, someone must enable build pipelines in your instance of {ProductName}. At the time of publication, this process includes configuring a smee channel, to listen for users' pull requests, and creating a GitHub App, so {ProductName} can access those PRs. The first procedure in this document explains how to perform these tasks. 
+
+Once build pipelines are enabled, users can create their applications in {ProductName}. The second procedure below describes how to create an application. 
+
+== Enabling build pipelines for your instance of {ProductName}:
+
+.Prerequisites:
+
+* Install link:https://github.com/konflux-ci/konflux-ci/tree/main[{ProductName}]
+* Read/write access to your {ProductName} instance’s repository
+
+.Procedure:
+
+. Start a new link:https://smee.io/[smee] channel.
+. In the repository for your instance of {ProductName}, edit the file /smee/smee-client.yaml. Replace `<smee-channel>` with the webhook proxy URL from the previous step. 
+. Create a GitHub App according to link:https://pipelinesascode.com/docs/install/github_apps/#manual-setup[these Pipelines as Code instructions]. For the *Webhook URL* field, use the webhook proxy URL for your smee channel.
```

Comment:
This is intended for the platform engineer to complete while users would complete later ones, right?